doc/api/vm.md

```diff
@@ -637,6 +637,31 @@ console.log(globalVar);
 // 1000
 ```
 
+## vm.compileFunction(code[, params][, options])
```

Comment:
Just realizing now that `params` is mandatory if one wants to pass `options`. If that is the desired behavior, the signature should be like this:

```md
## vm.compileFunction(code[, params[, options]])
```